presto-main/src/test/java/com/facebook/presto/sql/planner/optimizations/TestReorderWindows.java

```diff
@@ -339,7 +339,13 @@ private void assertUnitPlan(@Language("SQL") String sql, PlanMatchPattern patter
                                 new GatherAndMergeWindows.SwapAdjacentWindowsBySpecifications(0),
                                 new GatherAndMergeWindows.SwapAdjacentWindowsBySpecifications(1),
                                 new GatherAndMergeWindows.SwapAdjacentWindowsBySpecifications(2))),
-                new PruneUnreferencedOutputs());
+                new PruneUnreferencedOutputs(),
+                new IterativeOptimizer(
+                        new RuleStatsRecorder(),
+                        getQueryRunner().getStatsCalculator(),
+                        getQueryRunner().getEstimatedExchangesCostCalculator(),
+                        ImmutableSet.of(
+                                new RemoveRedundantIdentityProjections())));
         assertPlan(sql, pattern, optimizers);
```

Comment:
Not in ported PR.

Change this, because the planner will add one more project to cast the sortkey to the same type as the offset expression. While in this test, it only contains a minimum number of rules, which will fail to remove this additional project node.

I've checked queries end to end with the full plan optimizer list, this additional project node will be removed, hence not a problem